grafana-plugin/src/models/outgoing_webhook/outgoing_webhook.ts

```diff
@@ -43,18 +46,34 @@ export class OutgoingWebhookStore extends BaseStore {
 
   @action
   async updateItem(id: OutgoingWebhook['id'], fromOrganization = false) {
-    const response = await this.getById(id, false, fromOrganization);
+    let outgoungWebhook;
+
+    try {
+      outgoungWebhook = await this.getById(id, true, fromOrganization);
+    } catch (error) {
+      if (error.response.data.error_code === 'wrong_team') {
+        outgoungWebhook = {
```

Comment:
spelling mistake in variable name
```suggestion
    let outgoingWebhook;

    try {
      outgoingWebhook = await this.getById(id, true, fromOrganization);
    } catch (error) {
      if (error.response.data.error_code === 'wrong_team') {
        outgoingWebhook = {
```